Index: ../../frontend/src/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from \"react\";\r\nimport ReactDOM from \"react-dom/client\";\r\nimport { RecoilRoot } from \"recoil\";\r\n\r\nimport App from \"./App\";\r\nimport reportWebVitals from \"./reportWebVitals\";\r\n\r\nconst root = ReactDOM.createRoot(document.getElementById(\"root\"));\r\nroot.render(\r\n\r\n  // <React.StrictMode>\r\n    <RecoilRoot>\r\n      <App />,\r\n    </RecoilRoot>\r\n  // </React.StrictMode>,\r\n);\r\n\r\n// If you want to start measuring performance in your app, pass a function\r\n// to log results (for example: reportWebVitals(console.log))\r\n// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals\r\nreportWebVitals();\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../frontend/src/index.js b/../../frontend/src/index.js
--- a/../../frontend/src/index.js	
+++ b/../../frontend/src/index.js	
@@ -7,11 +7,10 @@
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-
   // <React.StrictMode>
-    <RecoilRoot>
-      <App />,
-    </RecoilRoot>
+  <RecoilRoot>
+    <App />
+  </RecoilRoot>,
   // </React.StrictMode>,
 );
 
